### DISCUSSION TITLE
Theme breaks component's default visual style.

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#1096] [PrimeVue]
I tried to change theme
 **from**: _import 'primevue/resources/themes/saga-green/theme.css';_ 
 **to**: _import 'primevue/resources/themes/md-light-indigo/theme.css';_
 
 indigo theme breaks components' default style. For example please take a look on the screenshots below to see the difference. THe first one is **md-indigo theme** and the second one is **saga-green theme.**
 
 **Addition to this, can I also change the Theme's default primary color globally in Vue 3 from main.js file?** 
 
![light-indigo-theme](https://github.com/primefaces/community/assets/74812496/7b2f8d37-6e56-4189-a14a-a3cd341f0672)
![light-green-theme](https://github.com/primefaces/community/assets/74812496/9427f537-7271-41fa-a157-b59ff5d7a1a8)

**Comment by tugcekucukoglu** [comment]
I cannot replicate the issue. Do you have a stackblitz link?
